### PR TITLE
Show empty state for no repo

### PR DESCRIPTION
**Pull request by codemonkey800**
## Description

#890

Shows empty state for maintenance section if the repo is not defined or a non-GitHub repo.

## Demos

image

### CODE REVIEW

File: frontend/i18n/en/activity.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
   "noData": {
     "allData": "No data could be fetched for this plugin. <hubLink href='$t(learnMoreLink)'>Learn more.</hubLink>",
     "monthly": "There is not yet enough data to display for this plugin",
-    "preview": "Activity not available for previews"
+    "preview": "Activity not available for previews",
+    "noCodeRepo": "Metrics not available, this plugin does not have a code repository specified."
```
Comment (klai95) [inline frontend/i18n/en/activity.json:14]: nit: I see that sometimes the text has `.`, and sometimes it does not. Curious as to why?

Comment (klai95) [reply frontend/i18n/en/activity.json:14]: Makes sense! Yeah definitely more of a question out of curiosity than one that requires a change!

**Review by Janeece — APPROVED** [review]
LGTM! 🙌

**Review by klai95 — APPROVED** [review]

File: frontend/i18n/en/activity.json (hunk 1 above)
Comment (Nadalyn-CZI) [reply frontend/i18n/en/activity.json:14]: If it is a complete sentence (as these are) it should be punctuated with ." at the end.  The comma is not needed. 

I suggest we enhance the wording of 
 "noCodeRepo": "Metrics not available, this plugin does not have a code repository specified." 
to
 "noCodeRepo": "Metrics not available, this plugin does not have a code repository specified or the repository is not on GitHub."

**Pull request merged**